.gitignore

```diff
@@ -7,3 +7,6 @@
 nosetests.xml
 .idea/
 .pyenv*/
+
```

Comment:
Perhaps copy and paste gitignore from https://www.gitignore.io/api/python

Reply:
It doesn't seem to have
`venv/` and `venv2/`,..the things I wanted git to ignore

Reply:
The one concern I have with this is that someone else would want to add `venv3`, `.venv`, `pyenv` etc for example.

Perhaps, only add the generic `.venv` or `venv` on the gitignore file.

Not sure if what I am saying makes any sense.

Alternatively, use this generic gitignore: https://www.gitignore.io/api/cmake,python,dotenv,virtualenv,sublimetext,visualstudiocode